aspnetcore/tutorials/min-web-api.md

```diff
@@ -0,0 +1,341 @@
+
+This tutorial teaches the basics of building a miminal web API with ASP.NET Core. For a tutorial on creating a web API project that contains more features, see [Create a web API](xref:tutorials/first-web-api)
+
+## Overview
+
+This tutorial creates the following API:
+
+|API | Description | Request body | Response body |
+|--- | ---- | ---- | ---- |
+|`GET /api` | Browser test, "Hello World" | None | `Hello World!`|
+|`GET /todoitems` | Get all to-do items | None | Array of to-do items|
+|`GET /todoitems/{id}` | Get an item by ID | None | To-do item|
+|`POST /todoitems` | Add a new item | To-do item | To-do item |
+|`PUT /todoitems/{id}` | Update an existing item &nbsp; | To-do item | None |
+|`DELETE /todoitems/{id}` &nbsp; &nbsp; | Delete an item &nbsp; &nbsp; | None | None|
+
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+[!INCLUDE[](~/includes/net-prereqs/6/net-prereqs-vs22-6.0.md)]
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/net-prereqs-vsc-6.0.md)]
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+[!INCLUDE[](~/includes/net-prereqs-mac-6.0.md)]
+
+---
+
+## Create an empty web project
+
+# [Visual Studio](#tab/visual-studio)
+
+* From the **File** menu, select **New** > **Project**.
+* Select the **ASP.NET Core Empty** template and select **Next**.
+* Name the project *TodoApi* and select **Next**.
+* In the **Additional information** dialog, select **.NET 6.0 (Preview)** and select **Create**.
+ 
+<!-- 
+![VS new project dialog](min-web-api/_static/5/vs.png)
+-->
+
+<!-- Move this later since we don't need it now -->
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* Open the [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal).
+* Change directories (`cd`) to the folder that will contain the project folder.
+* Run the following commands:
+
+   ```dotnetcli
+   dotnet new web -o TodoApi
+   cd TodoApi
+   code -r ../TodoApi
+   ```
+
+* When a dialog box asks if you want to add required assets to the project, select **Yes**.
+
+  The preceding commands:
+
+  * Creates a new web API project and opens it in Visual Studio Code.
+  * Adds the NuGet packages which are required in the next section.
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+* Select **File** > **New Solution**.
+
+`  ![macOS New solution](first-web-api-mac/_static/sln.png)`
+
+* In Visual Studio for Mac earlier than version 8.6, select **.NET Core** > **App** > **API** > **Next**. In version 8.6 or later, select **Web and Console** > **App** > **API** > **Next**.
+
+`  ![macOS API template selection](first-web-api-mac/_static/api_template.png)`
+
+* In the **Configure the new ASP.NET Core Web API** dialog, select the latest .NET Core 5.x **Target Framework**. Select **Next**.
```

Comment:
Just checking should this be **Configure the new ASP.NET Web ** dialog, select the latest .NET Core 6.x **Target Framework**. Select **Next**. ?  I don't have a mac to test this on

Reply:
Please hold off reviews of Mac CLI,haven't even started that yet